src/sdk/model/advanced-backups/recovery/__json__/search-vdc-recoverable-files-and-folders-filters-json.ts

```diff
@@ -8,6 +8,21 @@ export interface SearchVdcRecoverableFilesAndFoldersFiltersJson {
   end_time?: number;
   search?: string;
   backup_group_uids?: Array<string>;
+  backup_run_uids?: Array<string>;
   vm_uuids?: Array<string>;
   file_type: FileType;
 }
+
+/**
+ * Search vDC recoverable files and folders filters request params interface.
+ * These params are required to be in camelCase vs snake_case.
+ */
+export interface SearchVdcRecoverableFilesAndFoldersFiltersParams {
```

Comment:
If you're creating this interface because the request requires them to be camel case why don't we get rid of the snake_case json interface? Looks like that won't ever be used and renders useless?

Reply:
Yep, snake case not really needed. This was what my question about snake vs. camelCase from a while ago pertained to. Updated now.